src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="body">
+    <div class="stopwatch">
+      <div class="innerD"></div>
```

Comment:
The class name 'innerD' does not follow BEM methodology. Consider renaming it to follow the BEM naming convention, such as 'stopwatch__inner'.